sdk/digitaltwins/azure-digitaltwins-core/src/main/java/com/azure/digitaltwins/core/DigitalTwinsAsyncClient.java

```diff
@@ -1190,4 +1201,141 @@ public HttpPipeline getHttpPipeline() {
+    {
+        return this.protocolLayer.getEventRoutes().addWithResponseAsync(eventRouteId, eventRoute, context);
+    }
+
+    /**
+     * Get an event route.
+     * @param eventRouteId The Id of the event route to get.
+     * @return The retrieved event route.
+     */
+    @ServiceMethod(returns = ReturnType.SINGLE)
+    public Mono<EventRoute> getEventRoute(String eventRouteId)
+    {
+        return getEventRouteWithResponse(eventRouteId)
+            .map(Response::getValue);
+    }
+
+    /**
+     * Get an event route.
+     * @param eventRouteId The Id of the event route to get.
+     * @return A {@link Response} containing the retrieved event route.
+     */
+    @ServiceMethod(returns = ReturnType.SINGLE)
+    public Mono<Response<EventRoute>> getEventRouteWithResponse(String eventRouteId)
+    {
+        return withContext(context -> getEventRouteWithResponse(eventRouteId, context));
+    }
+
+    Mono<Response<EventRoute>> getEventRouteWithResponse(String eventRouteId, Context context)
+    {
+        return this.protocolLayer.getEventRoutes().getByIdWithResponseAsync(eventRouteId, context);
+    }
+
+    /**
+     * Delete an event route.
+     * @param eventRouteId The Id of the event route to delete.
+     * @return An empty mono.
+     */
+    @ServiceMethod(returns = ReturnType.SINGLE)
+    public Mono<Void> deleteEventRoute(String eventRouteId)
+    {
+        return deleteEventRouteWithResponse(eventRouteId)
+            .flatMap(voidResponse -> Mono.empty());
+    }
+
+    /**
+     * Delete an event route.
+     * @param eventRouteId The Id of the event route to delete.
+     * @return A {@link Response} containing an empty mono.
+     */
+    @ServiceMethod(returns = ReturnType.SINGLE)
+    public Mono<Response<Void>> deleteEventRouteWithResponse(String eventRouteId)
+    {
+        return withContext(context -> deleteEventRouteWithResponse(eventRouteId, context));
+    }
+
+    Mono<Response<Void>> deleteEventRouteWithResponse(String eventRouteId, Context context)
+    {
+        return this.protocolLayer.getEventRoutes().deleteWithResponseAsync(eventRouteId, context);
+    }
+
+    /**
+     * List all the event routes that exist in your digital twins instance.
+     * @return A {@link PagedFlux} that contains all the event routes that exist in your digital twins instance.
+     *         This PagedFlux may take multiple service requests to iterate over all event routes.
+     */
+    @ServiceMethod(returns = ReturnType.COLLECTION)
+    public PagedFlux<EventRoute> listEventRoutes()
+    {
+        return listEventRoutes(new EventRoutesListOptions());
+    }
+
+    /**
+     * List all the event routes that exist in your digital twins instance.
+     * @param options The optional parameters to use when listing event routes. See {@link EventRoutesListOptions} for more details
+     *                on what optional parameters can be set.
```

Comment:
There is some strange spacing in the comments.

Reply:
I'll fix these, sure